TFT/src/User/API/interfaceCmd.c

```diff
@@ -609,12 +618,11 @@ void sendQueueCmd(void)
           if (cmd_seen('R'))
           {
             setPrintRemainingTime((cmd_value() * 60));
-            setM73_presence(0);  // disable parsing remaning time from gCode comments
```

Comment:
Oopsie! Here should be `setM73_presence(true)`.
I do not know what happened here because I never use numeric values for boolean, always `true` or `false`.

Reply:
> it was always set to "false" in the code unless there is a case where it is set to "true" that I didn't see

See my comment in "interface.c" line 612. You just discovered a bug that I cannot explain how it slipped there.

Reply:
ok. reverted and fixed

Reply:
Thank you. Cheers!